### DISCUSSION TITLE
ClipboardCopy uses deprecated `document.execCommand` DOM API

### DISCUSSION THREAD

**Discussion by jkilzi** [patternfly/patternfly#5475] [Q&A] [answered]
It happens here:
https://github.com/patternfly/patternfly-react/blob/d026d3fbbcdaa8438312b71ebe59a2f232103b4d/packages/react-core/src/components/ClipboardCopy/ClipboardCopy.tsx#L19

Since this API is deprecated (and therefore could break at any moment), would you consider migrating this functionality to the [Clipboard API](https://developer.mozilla.org/en-US/docs/Web/API/Clipboard_API)?

**Comment by nicolethoen** [comment, marked as answer]
This is an old discussion, but FWIW, we have updated the ClipboardCopy component to use the new API :)
https://github.com/patternfly/patternfly-react/blob/main/packages/react-core/src/components/ClipboardCopy/ClipboardCopy.tsx#L15
